tests/test_eload_submission.py

```diff
@@ -24,7 +25,6 @@ def setUp(self) -> None:
         os.chdir(ROOT_DIR)
         self.eload = EloadPreparation(1)
```

Comment:
I guess we can either remove this or use `self.eload` in all the tests here... don't mind either way.

Reply:
Good catch. Change to actually made use of the object made in the setup